src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java

```diff
@@ -74,10 +74,10 @@ private GtfsRtFactory() {
     }
 
     public static GtfsRealtime.TripUpdate newTripUpdate(InternalMessages.TripCancellation cancellation, long timestampMs) {
-
+        final int gtfsRtDirection = StopEvent.pubtransDirectionToGtfsDirection(cancellation.getDirectionId());
```

Comment:
Could be renamed to `StopEvent.joreDirectionToGtfsDirection`?

Reply:
good point 👍